devdocs/orientation-guide.md

```diff
@@ -0,0 +1,120 @@
+![](images/teammates_heading.png)
```

Comment:
This picture does not appear properly... I think we have to either rename the actual file to `.png` (all small case) or change this to `.PNG` (all upper case)